graph/schema.resolvers.go

```diff
@@ -120,6 +120,60 @@ func (r *mutationResolver) SetCalorieGoal(ctx context.Context, input model.Calor
 	}, nil
 }
 
+func (r *mutationResolver) DeleteUser(ctx context.Context, id string) (*bool, error) {
+	// Delete the row in the users table with an ID matching the one passed in
+	query := fmt.Sprintf(`DELETE FROM users WHERE user_id='%s';`, id)
```

Comment:
It should actually be sufficient to execute only this query.

The `exercise_records` and `food_records` tables have a foreign key relationship with the `users` table that is configured with "CASCADE" behavior on delete. This means that when the user is deleted, any records in other tables that have its ID as the foreign key value will also be deleted - in short, when the user is deleted the rows that point at it will be, too.